.github/workflows/codecov.yml

```
@@ -10,14 +10,21 @@ jobs:
     runs-on: ubuntu-latest
 
+    strategy:
+      fail-fast: false
+      matrix:
+        # Lowest supported version (see requires-python) and the newest one, so
+        # that the declared floor stays honest.
+        python-version: ['3.9', '3.13']
+
     steps:
     - name: Checkout code
       uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)